ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -132,6 +134,14 @@ const styles = StyleSheet.create({
   }
 });
 
+const BONUS: Bonus = {
```

Comment:
Could it be moved inside `bonusVacanze/mock`?
So at the end we will remove everything inside that file and we will link with the real data

Reply:
Yes of course, I'll add another `subdir` to the tree

Reply:
@Undermaken may we move also the type `Bonus` to the same folder?

Reply:
Sure! everything you need and you haven't, place it in a some dir inside `bonusVacanze`